src/Integration/Symfony/DependencyInjection/DomainExtension.php

```diff
@@ -24,6 +24,10 @@ public function load(array $configs, ContainerBuilder $container)
         );
         $loader->load('services.yaml');
 
+        if ($container->getParameter('kernel.debug')) {
+            $loader->load('services_dev.yaml');
```

Comment:
By convention the files `_{env}` are loaded from the value of the application environment, not the `kernel.debug` parameter. This is a bit confusing IMO. But it's a good idea to load this feature only when the application is in `debug mode`

Reply:
I don't understand your point. Can you elaborate ?

Reply:
Oh, naming. OK. I rename `services_dev.yaml` into `services.debug.yaml`.
Seems right to you ?